js/lib/alert-bar.js

```diff
@@ -0,0 +1,16 @@
+var $ = require('jquery')
+var callData = require('./communityCall.json')
+
+function callTime () {
+  var now = new Date().getHours()
+  var startTime = parseInt(callData.time, 10)
+  var dayOfWeek = new Date().getDay()
+  var reminder = $('.alert-bar')
+  var announcement = $('.lh-title').append('<span>The IPFS community call will start at ' + callData.time + '. Join us <a href=' + callData.zoomLink + '>here</a></span>')
+  if ((startTime - now <= 2 && startTime - now >= 0) && (dayOfWeek === 1)) {
```

Comment:
Banner will disappear as soon as the call starts. Should we leave it up for the duration of the call? - maybe encode this information in the `communityCall.json`.

Reply:
HI @alanshaw! Eric here (Proto-noob).

What if the message changed to, "The IPFS community call is now in progress! [Join us here.](url)" once the call began, so latecomers could join?

Also, i would make the entire "Join us here." phrase a link.

Reply:
> What if the message changed to, "The IPFS community call is now in progress! Join us here." once the call began, so latecomers could join?

I like that suggestion. That being said, the link to landing page will still be the same so that participants will know that they're joining a live call.